lerna.json

```diff
@@ -1,7 +1,4 @@
 {
-  "lerna": "2.0.0-rc.5",
-  "packages": [
```

Comment:
When the packages patterns matches the workspaces pattern it can safely be omitted. Lerna understands your npm workspaces configuration